### PR TITLE
HTML Meta tag

### LINKED ISSUE #6052: Frontend_Web_Development_HTML_CSS_JS : HTML meta tag
### Description

Documentation on HTML meta tag

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by palak1101**
<hr>

## Description 📜

- What is HTML Meta tag?
-  Key points on meta tag
-  Advantages of using meta tag
-  Attributes
-  Example with output
-  Supporting Browsers 

Fixes #6052

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by code-reaper08** [comment]
> @code-reaper08 I've committed suggested changes, please check it.
> 
> Also, about the deleted file Web_Development/FrontEnd/HTML: Search Engine Optimization Basics.md, I didn't do it. It got deleted while switching branch to Frontend_Web_Development_HTML_CSS_JS , this particular file was reverted because of which I wasn't able to switch branch.
> 
> Can you tell me how to undo deletion of this file?? Shall I simply delete this Seach Engine Optimization Basics.md file from PR.

Hello @palak1101 
You can't undo deletion, it'll be better if you create a new PR, with clean commits. Be careful and check twice, because commits can't be undone.

Also please mention this old PR in your new PR.

File: Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+ <meta attribute-name="value">
+ ```
+
+<br>
+
+## Visiblity of metadata on browsers-
+
+![image](https://user-images.githubusercontent.com/75716335/139000776-830f91e1-fad9-4db0-82e1-3f2fc43525ad.png)
+
+Source : Google Images
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md:20]: Embed the link of the image in the text.

File: Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- **content:** This attribute is used to specify properties value.
+- **charset:** This attribute is used to specify a character encoding for an HTML file.
+<br>
+
+>## **Note:**
+>The ```<meta>``` tag also accepts Global Attributes as well as Event Attributes in HTML.
+
+<br>
+
+## **Following are some specific syntaxes of meta tag which shows the different uses of meta Tag :**
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md:62]: Remove colon from the end.

File: Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+## **Supporting Browsers :**
+Element - ```<meta>``` tag 
+<br>   
+
+| **Chrome** |  **IE** | **Firefox** | **Oracle** | **Safari** |
+
+<br>
+
+---
+*Thanks for reading !*
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md:181]: Remove these lines.

**Review by kashishahuja2002 — COMMENTED** [review]
image

Plagiarism detected, refactor it.

No need for bold in headings.
Remove all the br tags.
Remove all the binary files and directly add images into the md file.

**Pull request closed without merging**